settings.py

```diff
@@ -496,6 +496,23 @@ def validate(cls, path: str) -> None:
                 raise ValueError(f"File hash does not match for {path}")
 
 
+class NESRomPath(UserFilePath):
+    """Special UserFilePath that ignores a non-optional variable header when validating. Using this will require
```

Comment:
The reason why we don't use doc string (in snes) is because that would show up in the host.yaml